helix-term/src/commands.rs

```diff
@@ -2994,6 +2995,11 @@ fn jumplist_picker(cx: &mut Context) {
     cx.push_layer(Box::new(overlaid(picker)));
 }
 
+fn changed_file_picker(cx: &mut Context) {
+    let picker = ui::changed_file_picker(cx.editor);
```

Comment:
The file picker is only defined in the `ui` mod because it's reused in a few places across modules. Let's define this picker here within the command like the other pickers